lib/shared/components/alerts/SafeAppAlert.tsx

```diff
@@ -0,0 +1,30 @@
+import { HStack } from '@chakra-ui/react'
+import { BalAlert } from './BalAlert'
+import { BalAlertButtonLink } from './BalAlertButtonLink'
+import { BalAlertContent } from './BalAlertContent'
+import { useWalletConnectMetadata } from '@/lib/modules/web3/useWalletConnectMetadata'
+import { useBreakpoints } from '../../hooks/useBreakpoints'
+
+export function SafeAppAlert() {
+  const { isMobile } = useBreakpoints()
+  const { isSafeAccountViaWalletConnect } = useWalletConnectMetadata()
+  if (isSafeAccountViaWalletConnect && !isMobile) {
+    return <BalAlert content={<Title />} status="info" />
```

Comment:
Only a comment, happy to keep as is.
```suggestion
    return <BalAlert content={<Content />} status="info" />
```